app/models/review.rb

```diff
@@ -1,16 +1,8 @@
 class Review <ActiveRecord::Base
-  belongs_to :user
-  belongs_to :video
+  include BelongsToUserable
```

Comment:
I am not sure, if this refactoring is the right path. This way creates another layer of indirection for a simple line.

Note: I just looked at those `BelongsTo....` modules and I would not think that some methods would be moved there.